public/app/features/dashboard-scene/settings/VariablesEditView.tsx

```diff
@@ -119,21 +119,45 @@ export class VariablesEditView extends SceneObjectBase<VariablesEditViewState> i
     }
     this.setState({ editIndex: variableIndex });
   };
+
+  public onUpdate = (variable: SceneVariable) => {
```

Comment:
Add test

Reply:
already in place, right?? :)

Reply:
It is needed because of the type change, that needs to create a new scene object every time the type changes